doc/api/url.md

```diff
@@ -53,6 +53,7 @@ WHATWG URL's `origin` property includes `protocol` and `host`, but not
 Parsing the URL string using the WHATWG API:
 
 ```js
+const URL = require('url').URL
```

Comment:
Now that `URL` is global, this is not strictly required. This might be improved by explaining that distinction?

Reply:
I know it works global :)
I'm thinking out loud in this PR whether we want to promote that or not and yes to make one or the other more explicit so devs are aware.